app/controllers/submitted_applications_controller.rb

```diff
@@ -0,0 +1,43 @@
+class SubmittedApplicationsController < ApplicationController
+  before_action :check_crime_application_presence,
+                :present_crime_application, only: [:show]
+
+  def index
+    # TODO: to be implemented.
+    #
+    # My suggestion is we use this action to render the `submitted`
+    # and `returned` listings. The default being `submitted` (we
+    # scope the collection `CrimeApplication.submitted`), and optional
+    # if a "status" query param is provided, we can filter by
+    # `submitted` or `returned` to render one or the other list.
+    #
+    # Example of routes:
+    # - /submitted/applications (renders `submitted` status tab)
+    # - /submitted/applications?status=submitted (same as above)
+    # - /submitted/applications?status=returned (renders `returned` tab)
+    #
+    # We don't have yet the `returned` status, and all this is really
+    # a bit pointless as this controller will ONLY talk to the
+    # document store through an API, but we don't have any of that yet.
```

Comment:
We have the api/applications and api/applications/{uuid} endpoints if you'd like to use this for the time being.

There is a very basic api client on review at the moment: https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/main/app/lib/api_client.rb

Reply:
Yes but wouldn't that API client be reading the applications back from our own database? 😄 
But yes, at some point, we will have to start decoupling from ActiveRecord and using something that feels more like "we don't have this locally anymore".